cmd/vulnx/clis/auth.go

```
@@ -0,0 +1,353 @@
+		} else {
+			gologger.Info().Msgf("  Key: %s (ENV)", strings.Repeat("*", len(envApiKey)))
+		}
+
+		fmt.Println("\nWhat would you like to do?")
+		fmt.Println("  1) Test current API key")
+		fmt.Println("  2) Configure new API key (save to credential store)")
+		fmt.Println("  3) Cancel")
+		fmt.Print("\nChoice (1-3): ")
+
+		var choice string
+		fmt.Scanln(&choice)
```

golangci-lint (1.64.8)

Error return value of `fmt.Scanln` is not checked

(errcheck)